commit f1252e4dbe781f75d806ce0b990779548eeeb7a9
Author: Mark Wielaard <mark@klomp.org>
Date:   Tue May 3 17:48:55 2022 +0200

    config: Move the 2>/dev/null inside the sh -c '' quotes for profile.csh.
    
    csh/tcsh would warn about "Ambiguous output redirect" if not done inside
    the sh -c command.
    
    Fix-by: наб <nabijaczleweli@nabijaczleweli.xyz>
    
    https://bugzilla.redhat.com/show_bug.cgi?id=2080957
    
    Signed-off-by: Mark Wielaard <mark@klomp.org>

diff --git a/config/profile.csh.in b/config/profile.csh.in
index 012e243a..74c20c99 100644
--- a/config/profile.csh.in
+++ b/config/profile.csh.in
@@ -6,7 +6,7 @@
 
 if (! $?DEBUGINFOD_URLS) then
     set prefix="@prefix@"
-    set DEBUGINFOD_URLS=`sh -c 'cat "$0"/*.urls; :' "@sysconfdir@/debuginfod" 2>/dev/null | tr '\n' ' '`
+    set DEBUGINFOD_URLS=`sh -c 'cat "$0"/*.urls 2>/dev/null; :' "@sysconfdir@/debuginfod" | tr '\n' ' '`
     if ( "$DEBUGINFOD_URLS" != "" ) then
         setenv DEBUGINFOD_URLS "$DEBUGINFOD_URLS"
     else
